File: RP6/d9-1.java
import java.util.Scanner;

public class Ex9_V1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String texto;
        System.out.print("Digite uma palavra ou frase: ");
        texto = sc.nextLine();

        int contador = 0;

        texto = texto.toLowerCase();

        for (int i = 0; i < texto.length(); i++) {
            char letra = texto.charAt(i);

            if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u') {
                contador++;
            }
        }

        System.out.println("Quantidade de vogais: " + contador);
    }
}

File: LE6/d9-1.cpp
#include <iostream>
using namespace std;

int main() {
    string texto;
    int contador = 0;

    cout << "Digite uma palavra ou frase: ";
    getline(cin, texto);

    for (int i = 0; i < texto.length(); i++) {
        char letra = tolower(texto[i]);

        if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u') {
            contador++;
        }
    }

    cout << "Quantidade de vogais: " << contador << endl;

    return 0;
}